### PR TITLE
Add known issue console in 2.6

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/25389

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -114,6 +114,16 @@ When you create an Ansible Tower automation template with the console automation
...
 
+[#cannot-edit-namespace-bindings-for-cluster]
+=== Cannot edit namespace bindings for cluster
+//2.6:25389
+
+When you edit namespace bindings for a cluster set with the `admin` role or `bind` role, you might encounter the following error:
+
+`ResourceError: managedclustersetbindings.cluster.open-cluster-management.io "<cluster-set>" is forbidden: User "<user>" cannot create/delete resource "managedclustersetbindings" in API group "cluster.open-cluster-management.io" in the namespace "<namespace>".`
+
+To work around the issue, make sure you also have permission to create or delete a `ManagedClusterSetBinding` resource in the namespace you want to bind. The role bindings only allow you to bind the cluster set to the namespace.
```
Comment (KevinFCormier) [inline release_notes/known_issues.adoc]: Could we avoid the words "work around" and maybe say "resolve" instead? We usually use that phrase when the problem is a bug. Although we are adding this as a known issue, the issue that we intend to fix is that the UI should not show namespaces for which the user would encounter these errors. 

Also, will you open another PR for 2.5 once we finalize the wording on this one?

Comment (oafischer) [reply release_notes/known_issues.adoc]: Yes, I'll change it to resolve. I'll start a PR for 2.5 once this one is merged.

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -114,6 +114,16 @@ When you create an Ansible Tower automation template with the console automation
 
 *Note:* While you cannot create Ansible Tower templates from credentials with a host URL that contains an insecure protocol using the console, you can still manually create or patch a resource using an insecure protocol.
 
+[#cannot-edit-namespace-bindings-for-cluster]
+=== Cannot edit namespace bindings for cluster
```
Comment (cadawson) [inline release_notes/known_issues.adoc]: Should this be cluster set?

**Review by cadawson — APPROVED** [review]
Other than that question, it looks good. Thanks!

File: release_notes/known_issues.adoc (hunk 2 above)
Comment (KevinFCormier) [reply release_notes/known_issues.adoc]: Good catch!

File: release_notes/known_issues.adoc [hunk 3]
```diff
@@ -114,6 +114,16 @@ When you create an Ansible Tower automation template with the console automation
...
+//2.6:25389
+
+When you edit namespace bindings for a cluster set with the `admin` role or `bind` role, you might encounter the following error:
+
+`ResourceError: managedclustersetbindings.cluster.open-cluster-management.io "<cluster-set>" is forbidden: User "<user>" cannot create/delete resource "managedclustersetbindings" in API group "cluster.open-cluster-management.io" in the namespace "<namespace>".`
```
Comment (KevinFCormier) [inline release_notes/known_issues.adoc:123]: Also note that this error has text in angle brackets: `<cluster-set>`, `<user>`, and `<namespace>`

These are placeholders for the actual values. Not sure if we need to point that out, or say they might see an error similar to the following or something to that effect.

File: release_notes/known_issues.adoc (hunk 2 above)
Comment (oafischer) [reply release_notes/known_issues.adoc]: Good catch, yes. Fixed it, thanks!

File: release_notes/known_issues.adoc (hunk 3 above)
Comment (oafischer) [reply release_notes/known_issues.adoc:123]: In that case we can say something like "you might encounter an error that resembles the following message". I'll update it, thanks.

**Review by KevinFCormier — APPROVED** [review]

**Pull request merged**